.github/workflows/test-snapshotter.yml

```diff
@@ -0,0 +1,32 @@
+name: test-snapshotter
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "master"
+      - "[0-9]+.[0-9]+"
+    tags:
+      - "v*"
+  pull_request:
+
+jobs:
+  validate-dco:
+    uses: ./.github/workflows/.dco.yml
+
+  build-dev:
+    needs:
+      - validate-dco
+    uses: ./.github/workflows/.build-dev.yml
```

Comment:
`build-dev` is called from `test-graphdriver` and `test-snapshotter` which would make the cache racy.